themes/edge.edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edge.edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
+        border-color: $honor-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/honor.png');
+        }
+    }
+
+    &.mode-verified {
+        border-color: $verified-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/verified.png');
+        }
+    }
+
+    &.mode-professional {
+        border-color: $professional-certificate-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/professional-education.png');
```

Comment:
Reminder to self (and/or @andy-armstrong) to review sandbox and ensure that the cert mode is accessible.